weekly-mission(week2)/src/constants/errorMessage.js

```diff
@@ -0,0 +1,18 @@
+const ERROR_MESSAGE = Object.freeze({
+  EMAIL: {
+    BLANK: '이메일을 입력해 주세요.',
```

Comment:
지금 보니까 BLANK 보다는 EMPTY가 의미가 더 명확할 것 같아요!
```suggestion
    EMPTY: '이메일을 입력해 주세요.',
```